packages/components/src/components/page-overlay/page-overlay.tsx

```diff
@@ -37,14 +39,20 @@ const PageOverlay = ({
         window.location.href = platforms.tradershub_os.url ?? '';
     };
 
+    const redirectToP2P = () => {
+        window.location.href = platforms.p2p_v2.url ?? '';
```

Comment:
just a question, this will also redirect to staging p2p if the user is on staging app right?

Reply:
yes it will redirect to staging p2p